### ISSUE TITLE
Download items reappear when deleted.

### ISSUE THREAD

**Issue by artelse** [arranger1044/iSoul#40]
And this only happens when the file has encoding errors. So one deletes an item from downloads, switch to another pane and back to downloads and there the 'deleted' item reappears. Character encoding seems a tough problem at slsk.

**Comment by arranger1044** [comment]
Yep, the encoding issues come from the slsk network nature, but we could do something about it in the future, like making encoding switch on the fly for chat messages ie. Maybe even for filesystem reinterpretation too.
